### PR TITLE
Ensure that referencing lodash occurs via the versionist base directory and not the CWD

### PR DESCRIPTION
**Pull request by hedss**
…ry and not the CWD.

Connects to #53.

change-type: patch

### CODE REVIEW

File: versionist.conf.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var _ = require('lodash');
+var _ = require(`${__dirname}/node_modules/lodash`);
```
Comment (lurch) [inline versionist.conf.js]: Perhaps `${__dirname}/node_modules` could be stored in a 'global variable' that versionist itself supplies to `versionist.conf.js` ?

Comment (hedss) [inline versionist.conf.js]: Yes, possibly, but given I've currently a broken published version, I'd really like to fix it.

Comment (hedss) [inline versionist.conf.js]: Created #55 to ensure this doesn't vanish.

Comment (jviotti) [inline versionist.conf.js]: I suggest to make Versionist accept a function being returned from `versionist.conf.js`, and if it detects that, pass lodash directly to it, just for convenience. For example:

```js
// versionist.conf.js

module.exports = (utils) => {
    // utils.lodash.isEmpty()
    // utils.lodash.each()
    // etc ...
    return {
        // The usual configuration
    };
};
```

**Pull request merged**